controllers/useraccount/console_usersettings_test.go

```diff
@@ -0,0 +1,155 @@
+package useraccount
+
+import (
+	"context"
+	"fmt"
+	. "github.com/codeready-toolchain/member-operator/test"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestDeleteConsoleSettingObjects(t *testing.T) {
+	t.Run("Object found by name and deleted", func(t *testing.T) {
+		// given
+		ctx := context.Background()
+		cm := &corev1.ConfigMap{
+			TypeMeta: metav1.TypeMeta{Kind: "ConfigMap"},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "user-settings-johnsmith",
+				Namespace: UserSettingNS,
+			},
+		}
+		cl := fake.NewClientBuilder().WithObjects(cm).Build()
+
+		// when
+		err := deleteResource(ctx, cl, "johnsmith", &corev1.ConfigMap{})
+
+		// then
+		require.NoError(t, err)
+		// check that the configmap doesn't exist anymore
+		err = cl.Get(ctx, client.ObjectKey{Name: "user-settings-johnsmith", Namespace: UserSettingNS}, &corev1.ConfigMap{})
+		require.True(t, errors.IsNotFound(err))
```

Comment:
```suggestion
		AssertObjectNotFound(t, cl, UserSettingNS, "user-settings-johnsmith", &corev1.ConfigMap{})
```

Reply:
done in [45ad586](https://github.com/codeready-toolchain/member-operator/pull/568/commits/45ad586544a4cb59d77f4fe273b49369548ad134)